assets/src/js/admin/onboarding-wizard/components/continue-button/index.js

```diff
@@ -0,0 +1,28 @@
+// Import styles
+import './style.scss';
+
+const { __ } = wp.i18n;
+
+import { useStoreValue } from '../../app/store';
+import { goToStep } from '../../app/store/actions';
+
+import Button from '../button';
+
+const ContinueButton = () => {
+	const [ { currentStep }, dispatch ] = useStoreValue();
+
+	const chevron = <svg width="8" height="13" viewBox="0 0 8 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This looks like a separate component to me. Even if it isn't used elsewhere, I think the relative complexity of an SVG deserves its own component.

Reply:
Also, and SVG changes for different reasons than the component that uses it, which is another reason to separate it.

Reply:
This is good input/thinking, I've gone ahead and moved the Checkmark to its own component. Ref: https://github.com/impress-org/givewp/pull/4942/commits/f0711317f5af138dc87f4a287f62c127af7efa0f